setup.py

```diff
@@ -4,7 +4,7 @@
 
 from setuptools import find_packages, setup
 
-VERSION = (1, 3, 0)
+VERSION = (2, 0, 0)
```

Comment:
I'll handle that part as the versioning of bungiesearch is not tied to the version of elasticsearch.